cmd/broker/ingress/main.go

```diff
@@ -17,12 +17,18 @@ limitations under the License.
 package main
 
 import (
-	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"go.uber.org/zap"
+
+	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/signals"
 )
 
+// main creates and starts an ingress handler using default options.
+// 1. It listens on port 8080
+// 2. It reads "GOOGLE_CLOUD_PROJECT" env var for pubsub project. If the env var is empty, it retrieves project ID from
```

Comment:
We've previously used `PROJECT_ID` for this env var in other receive adapters. We should be consistent here.